OrbitCore/FunctionFramepointerValidator.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_FUNCTION_FRAMEPOINTER_VALIDATOR_H_
+#define ORBIT_CORE_FUNCTION_FRAMEPOINTER_VALIDATOR_H_
+
+#include <capstone/capstone.h>
+
+// Provide utilities to check whether a function was compiled with
+// "-fno-omit-frame-pointer (-momit-leaf-frame-pointer)". The latter one is
+// optional.
+// The validator checks the functions Prologue and also whether an Epilogue
+// exists.
+class FunctionFramepointerValidator {
+ public:
+  FunctionFramepointerValidator(csh handle, const uint8_t* code,
+                                size_t code_size);
+  virtual ~FunctionFramepointerValidator();
+
+  // FunctionFramepointerValidator is neither copyable nor movable.
+  FunctionFramepointerValidator(const FunctionFramepointerValidator&) = delete;
+  FunctionFramepointerValidator& operator=(
+      const FunctionFramepointerValidator&) = delete;
+  FunctionFramepointerValidator(FunctionFramepointerValidator&&) = delete;
+  FunctionFramepointerValidator& operator=(
+      const FunctionFramepointerValidator&&) = delete;
```

Comment:
Remove this const as it's an rvalue-ref.

Reply:
Okay. Thanks.